<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>854</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>778</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1224</width>
    <height>954</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Postpancreaticoduodenectomy sepsis risk stratification</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/title.png</normaloff>img/title.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 148, 200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>580</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 87 9pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string>Prediction accuracy:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Accuracy">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Gender">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LiverDisease">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>520</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OperativeTime">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PancreaticTexture">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PreoperativeBiliaryStent">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Gender (male=1/female=0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 87 9pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string>Is it possible to experience sepsis:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BUN">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ASAclass">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #F3ECDB;
font: 75 18pt &quot;Times New Roman&quot;;
color: #C8515C;</string>
    </property>
    <property name="text">
     <string>Postpancreaticoduodenectomy sepsis risk stratification</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>ASA class</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Pancreatic texture (Intermediate or hard=1/soft=0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Preoperative biliary stent (yes=1/no=0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>160</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>BUN (mg/dl)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>410</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Operative approach (minimally invasive surgery=1/open surgery=0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>WBC (10^9/L)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Operative time (min)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OperativeApproach">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="WBC">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Click_For_Result">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 238, 225);
color: rgb(0, 0, 0);
font:8pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string>Click_For_Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 87 12pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string>Risk factors</string>
    </property>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>831</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Risk">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:8pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Risk</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
